### ISSUE TITLE
Convert Princeton mappings

### ISSUE THREAD

**Issue by simongray** [kuhumcst/DanNet#15]
The mapping between the old DanNet and Princeton WordNet are quite of messy. Rather than mapping directly to the instances of one WordNet dataset, they map to two different kinds of sense IDs using the `dns:eq_has_synonym` relation:

* "ENG20-0*2324230*-n"
  - this seems to be mapping to `wn:synset-sheep-noun-1` in WordNet 2.0 as it has the synsetID 10*2324230*.
* "sheep%1:05:00::" 
  - this seems to have already been covered by the above...? So why does it exist?

It may be beneficial to completely ignore these relations for now and revisit them once Sussi is done mapping relations, at which point they can be properly converted and included in the dataset.

**Comment by simongray** [comment]
Woops, this is a duplicate of #3. Closing.
